examples/basic_self_signed.yml

```diff
@@ -9,4 +9,4 @@
         ca: self-sign
 
   roles:
-    - certificate
+    - linux-system-roles.certificate
```

Comment:
This change looks unrelated to the subject of the PR, unless it is necessary to run the test

Reply:
This is actually part of PR #3 but it's also not required by that PR itself. It's there because running the tests I'm running the examples manually to ensure they work.